Resources/config/orm.xml

```diff
@@ -95,7 +95,7 @@
 
         <service id="doctrine.orm.configuration" class="%doctrine.orm.configuration.class%" abstract="true" public="false" />
 
-        <service id="doctrine.orm.entity_manager.abstract" class="%doctrine.orm.entity_manager.class%" abstract="true" />
+        <service id="doctrine.orm.entity_manager.abstract" class="%doctrine.orm.entity_manager.class%" abstract="true" lazy="true" />
```

Comment:
> You may want to set the attribute dynamically in the extension class depending on whether or not it is available.

"lazy" is simply no-op if the required components aren't installed.